tests/ui_tests/app/page/common/base_page.py

```diff
@@ -228,6 +230,10 @@ def wait_config_loaded(self):
         self.find_element(CommonLocators.socket, timeout=30)
         self.find_element(CommonLocators.profile, timeout=30)
 
+    def hover_element(self, locator: Locator):
```

Comment:
Maybe add docstring here?
Hover sounds like putting the mouse over an element, but `move_to_element` associates with scrolling.
A bit confusing for a method in `base_page`

Reply:
added